### ISSUE TITLE
Migrating profile to core repo

### ISSUE THREAD

**Issue by marcoscaceres** [Geonovum/respec#161]
As we continue to evolve ReSpec's code, I realize that it puts somewhat of a burden on the Geonovum folks to keep updating. I'm wondering if we should consider moving `src/geonovum` and the profile file over to the main W3C ReSpec repo, and then we could share ownership of the core modules?   

The advantages being:

 * Geonovum's profile would always be up to date with the latest "core".
 * No need to worry about merging the W3C ReSpec upstream. 
 * More eyes on bugs. 

The cons:

 * Some loss of control over distribution (but could deal with that, depending on what's needed by Geonovum's community). 
 * More stringent quality control requirements, particularly around testing (this might be an advantage) 

Just some thoughts... would like to hear what you folks think!

**Comment by lvdbrink** [comment]
It sounds good to me! @thijsbrentjens what do you think (since you are doing most of the merging these days)?

**Comment by thijsbrentjens** [comment]
I agree, would be a nice option. And adding tests is on our to do list too, so that's more a pro than a con. The loss of control over distribution won't be a big issue in practice I think.

**Comment by marcoscaceres** [comment]
Ok, that's great @lvdbrink @thijsbrentjens. I'll set a few things up on my end. 

If you have any outstanding issues in this repository, you can try to close them in the meantime.

About the testing, what we could do is added tests as we migrate each Geonovum module over. I'm available for review and can provide some guidance. 

If you think it would be helpful, we can set up call and show you how to write and run the tests.

**Comment by thijsbrentjens** [comment]
Thanks @marcoscaceres, that is great. A call would be helpful (we'll send you an email)
